app/controllers/concerns/verify_sp_attributes_concern.rb

```diff
@@ -1,10 +1,6 @@
 module VerifySpAttributesConcern
   def needs_completions_screen?
```

Comment:
another thought, we could remove this method entirely and just replace it with a check for the reason? don't even need the `.present?`

Reply:
I think this is used in a _bunch_ of places and I kind of like it for readability, personally. Gonna merge unless you have strong objections.

Reply:
I don't feel that strongly. But just for reference, it's only used in 4 places! All we gotta do is remove the `?` at the end 

```
git grep -F "needs_completions_screen?" -- app 
app/controllers/application_controller.rb:    if needs_completions_screen?
app/controllers/concerns/verify_sp_attributes_concern.rb:  def needs_completions_screen?
app/controllers/concerns/verify_sp_attributes_concern.rb:    if needs_completions_screen?
app/controllers/sign_up/completions_controller.rb:      if needs_completions_screen?
app/controllers/sign_up/completions_controller.rb:      track_completion_event('agency-page') if needs_completions_screen?
```

Reply:
I agree with Zach, less indirection is good 🙂

Reply:
@zachmargolis So would you suggest renaming `needs_completions_screen_reason` to `needs_completions_screen`?

Reply:
No sorry my brain got scrambled, I think we should keep it as `_reason`, I'm proposing this:

```
git grep -l -F "needs_completions_screen?" | xargs perl -p -i -e 's/needs_completions_screen\?/needs_completions_screen_reason/g'
```

(plus some a bit of manual cleanup to remove the predicate method)

Reply:
ok that makes more sense. And no need to explicitly call `.present?`?

Reply:
I say no `.present?` is redundant, because the method returns nil when there is no reason

Reply:
works for me - I always question whether to add it or not to be explicit. Updated PR incoming!

Reply:
@zachmargolis your local branch must have been out of date, there were a few other places 😛. I also updated the concern specs to test the `reason` method instead.